[Staking]: clicking on a staking transaction crashes the app

![Screenshot from 2023-09-26 11-17-36](https://github.com/crypto-power/cryptopower/assets/25265396/dcf3f949-8ffc-4f57-800f-edbb40fb82af)

i clicked on the voted transaction and the app crashed, subsequently clicking on revoked and other staking transactions crashes the app

```
2023-09-26 11:15:01.807 [WRN] DCR: unable to getWallet info using ticket: 7bfdd587a9020734686ed3c40b19f7f7a96a20b8794565ea2d9fa014946cdff0 Error: item does not exist: no VSP info for ticket 7bfdd587a9020734686ed3c40b19f7f7a96a20b8794565ea2d9fa014946cdff0
2023-09-26 11:15:01.807 [ERR] UI: VSPTicketInfo error: item does not exist: no VSP info for ticket 7bfdd587a9020734686ed3c40b19f7f7a96a20b8794565ea2d9fa014946cdff0
2023-09-26 11:15:01.807 [WRN] DCR: unable to getWallet info using ticket: 7bfdd587a9020734686ed3c40b19f7f7a96a20b8794565ea2d9fa014946cdff0 Error: item does not exist: no VSP info for ticket 7bfdd587a9020734686ed3c40b19f7f7a96a20b8794565ea2d9fa014946cdff0
2023-09-26 11:15:01.807 [ERR] UI: VSPTicketInfo error: item does not exist: no VSP info for ticket 7bfdd587a9020734686ed3c40b19f7f7a96a20b8794565ea2d9fa014946cdff0
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x12e7c08]

goroutine 1228 [running]:
github.com/crypto-power/cryptopower/ui/page/transaction.(*TxDetailsPage).OnNavigatedTo.func1()
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/page/transaction/transaction_details_page.go:228 +0x2a8
created by github.com/crypto-power/cryptopower/ui/page/transaction.(*TxDetailsPage).OnNavigatedTo
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/page/transaction/transaction_details_page.go:216 +0x1ea
exit status 2
```

--- a/ui/page/transaction/transaction_details_page.go
+++ b/ui/page/transaction/transaction_details_page.go
@@ -214,20 +214,30 @@ func (pg *TxDetailsPage) OnNavigatedTo() {
 
 		if pg.wallet.TxMatchesFilter(pg.transaction, libutils.TxFilterStaking) {
 			go func() {
-				info, err := dcrImp.VSPTicketInfo(pg.transaction.Hash)
-				if err != nil {
-					log.Errorf("VSPTicketInfo error: %v", err)
-				}
-
 				pg.vspHost = values.String(values.StrNotAvailable)
+				pg.vspHostFees = values.String(values.StrNotAvailable)
+
+				var feeTxHash string
+				info, err := dcrImp.VSPTicketInfo(pg.transaction.Hash)
 				if info != nil {
 					pg.vspHost = info.VSP
+					feeTxHash = info.FeeTxHash
+				}
+				if err != nil {
+					log.Errorf("VSPTicketInfo error: %v", err)
+					return
 				}
 
-				pg.vspHostFees = values.String(values.StrNotAvailable)
-				feeTx, _ := pg.wallet.GetTransactionRaw(info.FeeTxHash)
+				if feeTxHash == "" {
+					return
+				}
+
+				feeTx, err := pg.wallet.GetTransactionRaw(feeTxHash)
 				if feeTx != nil {
 					pg.vspHostFees = pg.wallet.ToAmount(feeTx.Amount).String()
+				}
+				if err != nil {
+					log.Errorf("GetTransactionRaw error: %v", err)
 				}
 			}()
 		}

